docs/snippets/src/main.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+#! [import]
+import openvino.runtime as ov
+#! [import]
+
+#! [part1]
+core = ov.Core()
+model = ov.Model()
+compiled_model = ov.CompiledModel()
```

Comment:
I am not sure if this code is OK to show. Both Model and CompiledModel constructors are not valid. Model has to at least have two arguments passed as result and parameters. CompiledModel has no empty constructor at all, it can be only obtained from dedicated functions.

Reply:
Thanks, fixed